add-on/chrome/bootstrap.js

```diff
@@ -1480,10 +1478,22 @@ function shutdown(data, reason) {
   // Stop waiting for browser windows to open.
   wm.removeListener(WindowListener);
 
-  // unregister about: handlers
-  AboutLoop.conversation.unregister(); // eslint-disable-line no-undef
-  AboutLoop.panel.unregister(); // eslint-disable-line no-undef
-  AboutLoop.toc.unregister(); // eslint-disable-line no-undef
+  // XXX do we really want the global PMM here, since it (unnecessarily, I
+  // think) sends a message to the CIPMM (is there any alternative?)
```

Comment:
The way I understand it, the global PMM is the right thing to do. We could potentially scope it down to PPMMs and see if we can find out which ones are attached to which browsers (so for example about:loop* would only work in the panel, sidebar & toc areas), however I think we don't really need that limitation.